BackofficeBundle/Resources/public/coffee/tinyMCE/activateTinyMCE.coffee

```diff
@@ -1,7 +1,22 @@
 # ACTIVATE TINYMCE
+
 callback_tinymce_init = null
 
+doCallBack = (editor, view) ->
```

Comment:
All right :).

This question was really only for my own knowledge, but I will agree that we open a discussion on a refacto on this part.